profiles/base.yaml

```diff
@@ -103,6 +103,8 @@ dance_actions:
 - weave
 # how many times to retry a failed analyze when using Analyze in training_abilities
 combat_analyze_retry_count: 3
+# Empath setting for using script empathylink script for combat.
```

Comment:
💡 Does this need to be clearer on how true vs false alter the script's behavior, or would that be intuitive for empath players?

Reply:
I can make clearer. :)